viewing_party/party.py

```diff
@@ -0,0 +1,250 @@
+#WAVE_01
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+    if movie_title and genre and rating != None:
+        new_movie["title"]= movie_title   
+        new_movie["genre"]= genre
+        new_movie["rating"]= rating
+        return(new_movie)
+    else:
+        return None
```

Comment:
Consider first testing for the edge case, and then moving the main functionality of your function outside the `if/else`. This programming pattern is called a guard clause and can among other things enhance the readability of your code.
 
```suggestion
    if not movie_title and not genre and not rating:
        return None
        
    movie = {}
    movie["title"] = movie_title
    movie["genre"] = genre
    movie["rating"] = rating
```